src/GNDStk/keyword/test/keyword.test.cpp

```diff
@@ -160,24 +159,21 @@ SCENARIO("Testing GNDStk keyword") {
    // ------------------------
 
    WHEN("We access a tree using some built-in child_t and meta_t objects") {
-      // child::xml disambiguates vs. class xml
-      // meta::format disambiguates vs. enum class format
-
-      CHECK(tree(mixed::child::xml).metadata.size() == 2);
-      CHECK(tree(mixed::child::xml).children.size() == 0);
-      CHECK(tree(mixed::child::xml,version) == "1.0");
-      CHECK(tree(mixed::child::xml,encoding) == "UTF-8");
+      CHECK(tree(xml).metadata.size() == 2);
+      CHECK(tree(xml).children.size() == 0);
+      CHECK(tree(xml,version) == "1.0");
+      CHECK(tree(xml,encoding) == "UTF-8");
       CHECK(tree(covarianceSuite).metadata.size() == 4);
       CHECK(tree(covarianceSuite).children.size() == 3);
       CHECK(tree(covarianceSuite,projectile) == "n");
       CHECK(tree(covarianceSuite,target) == "Tm170");
       CHECK(tree(covarianceSuite,evaluation) == "ENDF/B-8.0");
-      CHECK(tree(covarianceSuite,mixed::meta::format) == 1.9);
+      CHECK(tree(covarianceSuite,format) == 1.9);
```

Comment:
I'm not sure we should have a default type of double for the format version of the GNDS file. I'd leave it as a string to be honest. BTW: this is merely an observation, no need to change it right now.

Reply:
Yes, I think you're right. The `mixed` (now `misc`) namespace was meant to contain meta_t and child_t objects equipped here and there with types that seemed reasonable, given what I saw in existing GNDS files. It always was, and it remains, basically an approximation of what might be reasonable types for different objects. It's good for testing, but will probably be deprecated eventually, as we develop (or auto-build from the JSON specs) more-official types.